src/shared/scrapers/US/NYT.js

```diff
@@ -25,26 +25,25 @@ const scraper = {
     const data = await fetch.csv(this.url, false);
 
     // FIXME when we roll out new TZ support!
-    const fallback = process.env.USE_ISO_DATETIME ? new Date(datetime.now.at('America/New_York')) : datetime.getDate();
-    let scrapeDate = process.env.SCRAPE_DATE ? new Date(`${process.env.SCRAPE_DATE} 12:00:00`) : fallback;
+    let scrapeDate = process.env.SCRAPE_DATE
+      ? new Date(`${process.env.SCRAPE_DATE} 12:00:00`)
+      : new Date(datetime.now.at('America/New_York'));
```

Comment:
Not sure I'd suggest using this yet, may be safer to stick with `datetime.getDate()` for the time being!